src/components/PopoverWithMeasuredContent.tsx

```diff
@@ -60,73 +25,85 @@ const defaultProps = {
  * anchor position.
  */
 
-function PopoverWithMeasuredContent(props) {
+function PopoverWithMeasuredContent({
+    popoverDimensions = {
+        height: 0,
+        width: 0,
+    },
+    anchorPosition,
```

Comment:
```suggestion
    anchorPosition = {},
```

Reply:
No need for this, since this parameter is required